### PR TITLE
LPD-32236 rename util package to helper

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
follow up pr from https://github.com/brianchandotcom/liferay-portal/pull/154496#issuecomment-2372432307

I'll use this to also make the step tracker visible for open orders

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/helper/CommerceOrderStepTrackerHelperImpl.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public class CommerceOrderStepTrackerHelperImpl
 
 	@Override
 	public List<StepModel> getCommerceOrderSteps(
-			CommerceOrder commerceOrder, Locale locale)
+		CommerceOrder commerceOrder, Locale locale, boolean showOpenStatus)
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/helper/CommerceOrderStepTrackerHelperImpl.java]: Shouldn't it be `showOpenStatuses`?

File: modules/apps/commerce/commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/helper/CommerceOrderStepTrackerHelper.java [hunk 2]
```diff
@@ -18,7 +18,7 @@
 public interface CommerceOrderStepTrackerHelper {
 
 	public List<StepModel> getCommerceOrderSteps(
-			CommerceOrder commerceOrder, Locale locale)
+		CommerceOrder commerceOrder, Locale locale, boolean showOpenStatus)
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-frontend-api/src/main/java/com/liferay/commerce/frontend/helper/CommerceOrderStepTrackerHelper.java]: Shouldn't it be `showOpenStatuses`?

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@andrea-ale-sbarra LGTM overall, just a small change, thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
Failures compared [here](https://testray.liferay.com/#/compare-runs/30964004/30997276/details?filter=%7B%22testrayCaseResultStatus1%22%3A%5B%22FAILED%22%5D%2C%22testrayCaseResultStatus2%22%3A%5B%22PASSED%22%5D%7D&filterSchema=compareRunsCases)

**Review by alessiorendina — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**